tests/test_cli.py

```diff
@@ -532,3 +532,29 @@ def test_run_read_from_stdin(self):
             'stdin:2:10: [error] syntax error: '
             'mapping values are not allowed here\n'))
         self.assertEqual(err, '')
+
+    def test_run_supress_warnings(self):
```

Comment:
`supress` → `suppress` 

or `test_run_no_warnings()`?

Reply:
Another typo (I'm not native English speaker also). I've changed it as you suggests, it's more clear.